openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -37,6 +36,7 @@ def get_api_client(api_config, student):
     return EdxRestApiClient(api_config.internal_api_url, jwt=id_token)
 
 
+# TODO: Move to program utilities. Will require updating tests.
```

Comment:
Is it too much to ask to do this refactoring within this PR? I rather not have "utils.py" to depend on this module, let alone the circular dependency